src/pieces/api/assets.py

```diff
@@ -156,56 +155,48 @@ def delete_asset_by_id(asset_id):
     except Exception as e:
         return f"Failed to delete {asset_id}"
 
-def update_asset(asset_id, application):
-    ## NOT CURRENTLY WORKING ##
 
+def reclassify_asset(asset_id, classification):
     asset_api = pos_client.AssetApi(api_client)
+    with open(extensions_dir) as f:
+        extension_mapping = json.load(f)
+        if classification not in extension_mapping:
+            show_error(f"Invalid classification: {classification}","Please choose from the following: \n "+", ".join(extension_mapping.keys()))
+            return
+        
+    try:
+        response = asset_api.asset_reclassify(asset_reclassification=
```

Comment:
Please try this with an OCR snippet. TO create an OCR snippet, copy and paste a screenshot of code into the desktop app.

OCR snippets are handled differently in the case of editing + reclassifying than normal assets. We might also need to make this its separate issue, unless you want to layer it into this PR.

Reply:
Okay thank u I did not know that there is a different way to classify the ocr snippet I will try it and tell lwt you know